### PR TITLE
POL-159 Rename Custom Policies to Policies

### PR DESCRIPTION
**Pull request by theute**

### CODE REVIEW

File: src/js/App/Header/InsightsAbout.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ export class InsightsAbout extends Component {
                 { name: 'Migration Analytics', path: 'apps/migration-analytics/app.info.json', version: 'N/A' },
                 { name: 'Automation Hub', path: 'apps/automation-hub/app.info.json', version: 'N/A' },
                 { name: 'Automation Analytics', path: 'apps/automation-analytics/app.info.json', version: 'N/A' },
-                { name: 'Custom Policies', path: 'apps/custom-policies/app.info.json', version: 'N/A' }
+                { name: 'Policies', path: 'apps/custom-policies/app.info.json', version: 'N/A' }
```
Comment (ryelo) [inline src/js/App/Header/InsightsAbout.js]: if you've updated your app's path from `custom-policies` to `policies`, you'll need to change the path here too.

Comment (theute) [reply src/js/App/Header/InsightsAbout.js]: Indeed ! Fixed, thanks !
image

**Comment by theute** [comment]
Looks like Travis is stuck ?

**Comment by karelhala** [comment]
Restarting travis

**Comment by theute** [comment]
@karelhala @ryelo I created #730 which is the same as this one, but it passed Travis, so I'm closing this one.

**Pull request closed without merging**